cmd/wio/toolchain/npm/semver/query.go

```diff
@@ -211,11 +211,11 @@ func parseCmpQuery(str string) *singleBound {
     loc := opMatch.FindStringIndex(str)
     opStr := str[loc[0]:loc[1]]
     verStr := strings.Trim(str[loc[1]:], " ")
-    return &singleBound{op: queryMap[opStr], ver: parseIncompl(verStr)}
+    return &singleBound{op: queryMap[opStr], ver: parseIncomplete(verStr)}
```

Comment:
if you can figure out what it's short for then that means the short name is good enough >:(

Reply:
lol GoLand complained so I switched 😢